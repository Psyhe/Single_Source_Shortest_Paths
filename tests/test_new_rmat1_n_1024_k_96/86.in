1024 924 933
924 4 156
925 514 60
926 80 64
928 352 18
928 26 119
928 11 116
928 4 236
928 33 53
928 64 186
928 256 196
928 24 108
928 2 151
928 128 158
928 29 255
929 72 77
929 70 211
929 386 130
929 67 20
929 644 218
929 0 60
930 537 10
930 96 174
930 265 113
930 544 225
930 256 4
930 834 13
930 0 97
931 16 120
932 320 53
932 786 14
933 66 241
