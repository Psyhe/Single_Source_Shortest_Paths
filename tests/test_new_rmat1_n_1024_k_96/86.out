158
65
72
1e+18
27
22
9
125
60
257
